vagrant_cloud.gemspec

```
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'excon', '~> 0.73'
   s.add_runtime_dependency 'log4r', '~> 1.1.10'
-  s.add_runtime_dependency 'rexml', '~> 3.2.5'
+  s.add_runtime_dependency 'rexml', '~> 3.3.0'
```

```suggestion
  s.add_runtime_dependency 'rexml', '~> 3.3'
```

We should allow all minor version bumps. (For a minimum patch version constraint, use `'~> 3.3', '>= 3.3.8'` rather than `'~> 3.3.8'`.)